libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/AztecEditorFragment.java

```diff
@@ -129,7 +129,7 @@
     private boolean mEditorWasPaused = false;
     private boolean mHideActionBarOnSoftKeyboardUp = false;
 
-    private AztecText title;
+    private org.wordpress.android.editor.SourceViewEditText title;
```

Comment:
Not a fan of using 2 different classes with the same name here, should we just rename the one in `org.wordpress.android.editor` to a more generic name since it's only a listening for IMEBack events?